project-base/storefront/package.json

```diff
@@ -55,17 +55,17 @@
         "redis": "4.0.4",
         "tailwind-merge": "1.9.1",
         "tinycolor2": "1.4.2",
-        "urql": "3.0.3",
+        "urql": "4.0.4",
         "uuid": "8.3.2",
         "wonka": "6.3.2",
         "yup": "0.32.9",
         "zustand": "4.3.6"
     },
     "devDependencies": {
-        "@graphql-codegen/cli": "^3.2.2",
-        "@graphql-codegen/fragment-matcher": "^4.0.1",
-        "@graphql-codegen/introspection": "^3.0.1",
-        "@graphql-codegen/typescript-operations": "^3.0.2",
+        "@graphql-codegen/cli": "^4.0.1",
+        "@graphql-codegen/fragment-matcher": "^5.0.0",
+        "@graphql-codegen/introspection": "^4.0.0",
+        "@graphql-codegen/typescript-operations": "^4.0.1",
```

Comment:
Let's put specific version

Reply:
I can see this is the current state, probably changed in a later commit.